src/Entities/Properties/Date.php

```diff
@@ -95,10 +136,18 @@ public function getStart(): DateTime
     }
 
     /**
-     * @return DateTime
+     * @return ?DateTime
      */
-    public function getEnd(): DateTime
+    public function getEnd(): ?DateTime
     {
         return $this->getContent()->getEnd();
     }
+
+    /**
+     * @return bool
+     */
+    public function hasTime(): bool
+    {
+        return $this->getContent()->hasTime();
```

Comment:
should we add null checks in functions like this?

Reply:
Yes! :) 
Done ✅